Add recipe for maces-game

pawelbx opened the pull request:
### Brief summary of what the package does

Another anagram puzzle game for Emacs.

### Direct link to the package repository

https://github.com/pawelbx/maces-game

### Your association with the package

I'm the maintainer.

### Relevant communications with the upstream package maintainer

None needed

### Checklist

Please confirm with `x`:

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses). 
- [x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [x] I've used [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] `M-x checkdoc` is happy with my docstrings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)

purcell commented:
Thanks! Sent you a few little fixes in https://github.com/pawelbx/maces-game/pull/1.

Other notes:
- Don't choose your own face colours. Instead, `:inherit` standard built-in faces, so that your game will work with all themes but will still allow customisation. Generally you should avoid setting `:height` on faces.
- Your buffer should be called `*maces-game*`, not `maces-game`. This is an Emacs convention for non-file buffer naming.
- Instead of `:files ("maces-game.el" "words.txt")` in the recipe, please use `:files (:defaults "words.txt")`
- Consider using the prefix `maces-game--` for internal identifiers, rather than `maces-game-`.

Hope that helps!

pawelbx commented:
Thanks for your feedback and the pull request. I merged your PR, but I had to make one small change, which I explain in that PR. 

Here are the updates I made:
- Changed the face colors to inherit standard built in faces (the height attribute is still modified however)
- Renamed buffer to` *maces-game*`
- Added `:defaults` to `:files` list
- Changes to prefix `maces-game--`

purcell commented:
Super, thanks! Merging now... welcome to MELPA!
